lib/src/helpers/fonts_loader.dart

```diff
@@ -82,6 +85,31 @@ Map<String, List<Future<ByteData>>> loadFontsFromFontsDir([Package? package]) {
   return fontFamilyToData;
 }
 
+/// Load fonts and icons to make sure they show up in golden tests.
+///
+/// To use it efficiently:
+/// * Create a flutter_test_config.dart file. See:
+/// https://api.flutter.dev/flutter/flutter_test/flutter_test-library.html
+/// * add `await loadAppFonts();` in the `testExecutable` function.
+Future<void> loadAppFonts() async {
+  TestWidgetsFlutterBinding.ensureInitialized();
+
+  final fontManifest = await rootBundle.loadStructuredData<Iterable<dynamic>>(
+    'FontManifest.json',
+    (string) async => json.decode(string),
+  );
+  final waitList = <Future<void>>[];
+  for (final Map<String, dynamic> font in fontManifest) {
+    final fontLoader = FontLoader(font['family']);
```

Comment:
I see you didn't add the function for `derivedFontFamily` here. Why did you miss it?
![Screenshot 2024-07-27 at 14 26 42](https://github.com/user-attachments/assets/56464f5f-b685-45b6-868a-dbea17cc6b20)

Reply:
Hi, I don't understand what you're refering to. Is `derivedFontFamily` a function from a package? Or is it a proposition to extract some logic?

If it is the first, can you link a doc link to the function, because I couldn't find it anywhere I looked

Reply:
It is here: [font_loader.dart](https://github.com/eBay/flutter_glove_box/blob/5af8ed5285d9fa261e5db054cbeab22814a93e32/packages/golden_toolkit/lib/src/font_loader.dart)
Method `loadAppFonts` is moved from `flutter_glove_box` project.

Reply:
@mjablecnik the derivedFontFamily is needed because golden_tool_kit include the font roboto, and it can cause conflict with other packages that also include roboto.

adpative_test doesn't include it, so we don't need it

Reply:
@T-moz Ok, thank you for your response. :)